client/src/pages/about/[slug].js

```diff
@@ -15,21 +16,19 @@ const CoworkerPage = ({ data, preview = false }) => {
     enabled: preview,
   })
   const post = filterDataToSingleItem(previewData, preview)
+  const customMediaType = { image: { ...post.heroImage }, type: 'image' }
 
   return (
     <PageWrapper footerDark>
       {preview && <ExitPreviewLink />}
       {post && (
-        <>
-          <CoworkerHero
-            title={post?.fullname ?? post.fullname}
-            heroImage={post?.heroImage ?? post.heroImage}
-          >
-            {post?.role ?? post.role}
-          </CoworkerHero>
-          <Medarbetare info={post ?? post} />
-        </>
+        <Hero
+          title={post.fullname && post.fullname}
+          subtitle={post.role && post.role}
+          mediaType={post.heroImage && customMediaType}
+        />
       )}
+      <Medarbetare info={post ?? post} />
```

Comment:
```suggestion
      <Medarbetare info={post} />
```
`post ?? post´ this basically says "post or post"